test/functional/setup.js

```diff
@@ -12,6 +12,7 @@ const API_LEVEL_MAP = {
   8.0: '26',
   8.1: '27',
   9: '28',
+  10: '29',
```

Comment:
is there a way to have this list autofilled?

Reply:
Maybe. I could see if there's a JSON endpoint that defines this.

Reply:
You could use https://www.npmjs.com/package/android-versions, but it'd need a PR for Q.